app/main/util/queries.py

```diff
@@ -0,0 +1,18 @@
+def build_sorting_orders(request):
```

Comment:
might be worth looking at the build sort order function @pgandhizaizi made? Maybe its not entirely appropriate for this route since its quite different but maybe theres some similarity

Reply:
Thanks, in the process of writing some tests for both filter & sort but will use the updated function.

Reply:
Hi, noticed  build_sorting_orders_consignment(request) function, it is not required as I wrote a generic sort function which will work for all views, please look at the function build_sorting_orders

Reply:
@pgandhizaizi That's been removed and it's now using the shared sorting func.